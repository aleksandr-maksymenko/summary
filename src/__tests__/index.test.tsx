import React from 'react'
import { shallow } from 'enzyme'
import Index from 'src/pages/oleksandr_maksymenko'

describe('Index pages', () => {
  it('Should return Index pages', () => {
    const wrapper = shallow(<Index />)
    expect(wrapper).toMatchSnapshot()
  })
})
