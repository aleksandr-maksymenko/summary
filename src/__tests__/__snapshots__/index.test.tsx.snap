// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index pages Should return Index pages 1`] = `ShallowWrapper {}`;
